prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -270,6 +272,9 @@ if [[ $AGGREGATOR_TASKS == FORWARD_TF || $AGGREGATOR_TASKS == ALL ]]; then
   if [[ $CALIB_ZDC_TDC == 1 ]]; then
     add_W o2-zdc-tdccalib-workflow
   fi
+  if [[ $CALIB_FT0_TIMEOFFSET == 1 ]]; then
+    add_W o2-calibration-ft0-time-offset-calib "--tf-per-slot $FT0_TIMEOFFSET_TF_PER_SLOT"
```

Comment:
Hi @afurs . 
As discussed via mail, here you could add your extra config option:
```
add_W o2-calibration-ft0-time-offset-calib "--tf-per-slot $FT0_TIMEOFFSET_TF_PER_SLOT" "FT0CalibParam.mNExtraSlots=0;FT0CalibParam.mRebinFactorPerChID[180]=4;"

Reply:
Hello @chiarazampolli , I added default configKeyValue as arg for add_W()